eth/filters/filter_system.go

```diff
@@ -450,7 +450,7 @@ func (es *EventSystem) eventLoop() {
 		es.chainSub.Unsubscribe()
 	}()
 
-	index := make(filterIndex)
+	index := make(filterIndex, LastIndexSubscription-1)
```

Comment:
Honestly I am not sure if it's worthwhile, performance is not critical here. Personally I will prefer the original one.

Reply:
I agree. Thanks for the review!